test/notebook_tests/test_notebooks.py

```diff
@@ -110,9 +111,13 @@ def test_ipynb(dir_path, notebook, s3_bucket, region):
             copyfile(notebook, dest_file)
             _rename_bucket(dest_file, s3_bucket)
             errors = _run_notebook(dir_path, dest_file)
-            assert errors == [], "Errors found in {}\n{}".format(
-                notebook, [errors[row]["evalue"] for row in range(len(errors))]
-            )
+            acceptable_error_names = ["DeviceOfflineException"]
```

Comment:
This is good. Thanks. Only one question, this won't raise a sev2 when device is offline when it is not supposed to be, do we want that behavior?

Reply:
Yes, exactly. The assumption is we have HE alarms to catch devices being offline and don't need to rely on notebooks for that. However, the hole it leaves that I am concerned about is if a distinct error manifests as a DeviceOfflineException (maybe a breaking change to getDevice or something). Let's validate with Christian before merging this.